test_util/src/lib.rs

```diff
@@ -1835,21 +1835,16 @@ pub fn pattern_match(pattern: &str, s: &str, wildcard: &str) -> bool {
     if i == 0 {
       continue;
     }
-    dbg!(part, i);
```

Comment:
I've found this helpful though. Why remove it?

Reply:
I found it overly noisy if integration tests are failing. I'm fine with keeping it if you prefer.